voluncheer/profiles/urls.py

```diff
@@ -2,15 +2,16 @@
 from django.urls import path
 
 from profiles.views.home import home
-from profiles.views.profile import ProfileView
+from profiles.views.profile import *
```

Comment:
Please enumerate the actual imports rather than using a wild card. It makes identifying what is used where much easier.